source/FreeRTOS_IP_Utils.c

```diff
@@ -434,7 +448,7 @@ static BaseType_t prvChecksumProtocolChecks( size_t uxBufferLength,
             ( uxBufferLength < ( ipSIZE_OF_ETH_HEADER + pxSet->uxIPHeaderLength + ipSIZE_OF_ICMPv4_HEADER ) ) )
         {
             pxSet->usChecksum = ipINVALID_LENGTH;
-            xReturn = 9;
+            xReturn = 10;
```

Comment:
What is the difference between 9 and 10 here

Reply:
That's a debug variable. It's recorded in a debug variable called [xLocation](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/blob/c555c2c1bb511ce738c4fe8972141bc11594562b/source/FreeRTOS_IP_Utils.c#L1089) to let us know the real reason of failure.